<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoDialog</class>
 <widget class="QDialog" name="PhotoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование фото</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="previewLayout">
       <item>
        <widget class="QLabel" name="photoPreview">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>300</height>
          </size>
         </property>
         <property name="text">
          <string>Превью</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="styleSheet">
          <string>background-color: #f0f0f0; border: 1px solid #ccc;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>Имя файла</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Описание:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="descriptionEdit">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dateLabel">
         <property name="text">
          <string>Дата и время:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit">
         <property name="displayFormat">
          <string>dd.MM.yyyy HH:mm</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tagsLabel">
         <property name="text">
          <string>Теги:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="tagsListWidget">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
